common/helpers.py

```diff
@@ -66,6 +66,11 @@ def find(iterable, pred):
             return x
 
 
+def pluck(obj, *keys):
```

Comment:
Not sure how stable/etc. `funcy` is, but I guess that's what I'd do if it's solid (or if there's something else that's solid).